modules/oe_subscriptions_anonymous/tests/src/Functional/SubscribeTest.php

```diff
@@ -163,7 +165,8 @@ public function testForm(): void {
     $assert_session->fieldExists($mail_label)->setValue('another@example.com');
     $assert_session->fieldExists($terms_label)->check();
     $assert_session->buttonExists('Subscribe me')->press();
-    $assert_session->statusMessageContains('A confirmation e-email has been sent to your e-mail address.', 'status');
+    $assert_session->statusMessageContains('A confirmation email has been sent to your email address', 'warning');
+    $assert_session->statusMessageContains('To confirm your subscription, please click on the confirmation link sent to your e-mail address.', 'warning');
```

Comment:
We need to somehow test that the message contains the html. We cannot use this.

Reply:
Added a function to check tag => content inside the warning message